shipment_advice/tests/common.py

```diff
@@ -118,32 +119,45 @@ def _create_move(cls, picking_type, product, quantity, group=False):
         move.picking_id.action_assign()
         return move
 
-    def _confirm_shipment_advice(self, shipment_advice, arrival_date=None):
+    @classmethod
+    def _confirm_shipment_advice(cls, shipment_advice, arrival_date=None):
         if shipment_advice.state != "draft":
             return
         if arrival_date is None:
             arrival_date = fields.Datetime.now()
         shipment_advice.arrival_date = arrival_date
         shipment_advice.action_confirm()
+
+    def confirm_shipment_advice(self, shipment_advice, arrival_date=None):
+        self._confirm_shipment_advice(shipment_advice, arrival_date)
         self.assertEqual(shipment_advice.state, "confirmed")
 
-    def _in_progress_shipment_advice(self, shipment_advice, dock=None):
-        self._confirm_shipment_advice(shipment_advice)
+    @classmethod
+    def _in_progress_shipment_advice(cls, shipment_advice, dock=None):
```

Comment:
I would have named it to reflect it is an action
```suggestion
    def progress_shipment_advice(cls, shipment_advice, dock=None):
```